{{cookiecutter.repo_name}}/Pipfile

```diff
@@ -3,20 +3,21 @@ url = "https://pypi.python.org/simple"
 verify_ssl = true
 
 [packages]
-Django = "<2.1"
-boto = "<2.47,>=2.46"
-dj-database-url = "==0.4.2"
-django-debug-toolbar = "<1.8,>=1.7"
-django-redis = "<4.8,>=4.7"
-django-rq = "<0.10,>=0.9"
-django-rq-wrapper = "<2.2,>=2.1"
-django-storages = "<1.6,>=1.5"
+django = "<2.2"
+boto = "< 2.50.0,>= 2.49.0"
+dj-database-url = "==0.5.0"
+django-debug-toolbar = "<1.12,>=1.11"
+django-redis = "<4.11.0,>=4.10.0"
+django-rq = "<1.4.0,>=1.3.0"
+django-storages = "<1.8,>=1.7"
+django-model-utils = "<3.2"
```

Comment:
Curious why this got added? (Not that I disagree, just frankly am not super familiar with the package) It doesn't appear we use it anywhere yet?

Reply:
@alexbarbato  I didn't mean to add that. Good catch I was moving packages from another project. 

Just an FYI its collection of different mixins and classes. We use it a lot for `TimeStampedModel`